_includes/api/en/4x/req-originalUrl.md

```diff
@@ -14,11 +14,13 @@ console.dir(req.originalUrl)
 // => '/search?q=something'
 ```
 
-In a middleware function, `req.originalUrl` is a combination of `req.baseUrl` and `req.path`, as shown in the following example.
+`req.originalUrl` available both in middleware and router objects. It is a
+combination of `req.baseUrl` and `req.path` values. It will also include query
+string in case it will be present in request. Consider following example:
```

Comment:
Just some minor proofreading/language smoothing:

```suggestion
`req.originalUrl` is available both in middleware and router objects. It is a
combination of `req.baseUrl`, `req.path`, and any query string included
in the request. Consider following example:
```

Reply:
Maybe we should really just say that it is the combination of `req.baseUrl` and `req.url`?